config/traject.rb

```diff
@@ -587,6 +587,9 @@
 # 993 - Endowment Information
 to_field 'endowment_info_note_ssim', extract_marc('993an')
 
+# 995 - Dedication Information
+to_field 'dedication_info_note_ssim', extract_marc('995acn3')
```

Comment:
```suggestion
to_field 'dedication_ssim', extract_marc('995acn3')
```

what do you think for a shorter field name?

Reply:
I'm totally fine with that. I guess I was just going off of the convention for other fields with the `_note` suffix, but I feel like that is kind of unnecessary. `dedication_ssim` is cool with me. 🆒

Reply:
yeah we just need the `_ssim` suffix, endowment field could have just been `endowment_info_ssim` but does not matter here. thanks for changing :)